anaconda.spec.in

```diff
@@ -149,6 +146,18 @@ Obsoletes: booty <= 0.107-1
 The anaconda-core package contains the program which was used to install your
 system.
 
+%package live
+Summary: Live installation specific files and dependencies
+BuildRequires: desktop-file-utils
+# live installation currently implies a graphical installation
+Requires: anaconda-gui = %{version}-%{release}
+Requires: usermode
+Requires: zenity
+
+%description live
+The anaconda-live package contains scripts, data and dependencies needed for
```

Comment:
Instead of `needed` the `required by` word sounds better to me.

Reply:
Changed.